src/components/Map.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import GoogleMapReact from 'google-map-react';
+
+const Map = () => {
+  const defaultProps = {
+    center: {
+      lat: 48.8566,
+      lng: 2.3522,
+    },
+    zoom: 11,
+  };
+
+  return (
+    <div className="map">
+      <h1>What's Near Me?</h1>
+      <div className="google-map">
+        <GoogleMapReact
+          bootstrapURLKeys={{ key: 'AIzaSyCbfV0IAdkkGv-9mmuAkUJNzCPPfGRO6v0' }}
+          defaultCenter={defaultProps.center}
+          defaultZoom={defaultProps.zoom}
+        ></GoogleMapReact>
```

Comment:
I'm not 100% sure, but the `GoogleMapReact` could probably be a self-closing tag, since it doesn't have any children right now. `<GoogleMapReact />` .

You _could_ also spread the `defaultProps` into the component `{...defaultProps}`. but I kind of like how explicit this is, too. Since the `key` is going to be a constant, it's probably a good idea to add that to the defaultProps, as well.

Reply:
Good point! We missed that. Fixed now 😄. Thank you for pointing these things out.